mods/ts/rules/defaults.yaml

```diff
@@ -520,6 +521,7 @@
 		RequiresCondition: criticalspeed
 		Modifier: 60
 	Carryable:
+	RevealOnFire:
```

Comment:
You'll need to add `deployed` to the `ArmamentNames` for the deployed tick tank.  You will also need to add this trait to `gaarty` which is still implemented as a building.